doc/source/data/doc_code/tensor.py

```diff
@@ -197,7 +197,7 @@ def cast_udf(block: pa.Table) -> pa.Table:
 from ray.data.datasource import ImageFolderDatasource
 
 ds = ray.data.read_datasource(
-    ImageFolderDatasource(), root="example://image-folder", size=(128, 128))
+    ImageFolderDatasource(), root="example://image-folders", size=(128, 128))
```

Comment:
```suggestion
    ImageFolderDatasource(), root="example://image-folder", size=(128, 128))
```

This shouldn't change right?

Reply:
Or rather, `image-folders/simple`?

Reply:
Ah yeah `"example://image-folders/simple"` is the right one. I think the doctests build was broken when this was merged.